### ISSUE TITLE
Gemma integration 

### ISSUE THREAD

**Issue by Isaina** [vegaluisjose/mlx-rag#4]
Hi

can you also implement gemma model to compare with llama

best regards

**Comment by AbeEstrada** [comment]
@Isaina open `query_vdb.py` and replace:

```diff
- 36 model, tokenizer = load("mlx-community/NeuralBeagle14-7B-4bit-mlx")
+ 36 model, tokenizer = load("mlx-community/quantized-gemma-2b-it")
```

**Comment by Isaina** [comment]
Can YOU also support mlx on Intel macos platform

**Comment by AbeEstrada** [comment]
@Isaina No, [mlx](https://github.com/ml-explore/mlx) was created for Apple silicon. If you want to use Intel try [llama.cpp](https://github.com/ggerganov/llama.cpp)

**Comment by Isaina** [comment]
Ok thanks for gemma information On 05.04.2024. at 18:50, Abe Estrada wrote:
                
            
            
                
                    From: "Abe Estrada" ***@***.***>Date: 5 April 2024To: "vegaluisjose/mlx-rag" ***@***.***>Cc: "Mention" ***@***.***>,"Isaina" ***@***.***>Subject: Re: [vegaluisjose/mlx-rag] Gemma integration  (Issue #4)
                
                
@Isaina no, mlx was created for Apple silicon
